Library/SPIClient.m

```diff
@@ -535,7 +535,8 @@ - (void)initiateRefundTx:(NSString *)posRefId
     });
 }
 
-- (void)acceptSignature:(BOOL)accepted {
+- (void)acceptSignature:(NSString *)posRefId
```

Comment:
the posRefId should already exist in `weakSelf.state.txFlowState` object. so I think  we can simply access it by `weakSelf.state.txFlowState.posRefId`